docs/source/guides/worklock_guide.rst

```diff
@@ -7,34 +7,307 @@ WorkLock Guide
+the following principles:
+
+ - All of the tokens held by WorkLock will be distributed.
+ - All bids must be greater than or equal to the minimum allowed bid.
+ - For each bid, the surplus above the minimum allowed bid is called the `bonus`; all bids are composed of a `base` bid (fixed minimum bid) and a `bonus` bid (variable amount).
+ - Each bidder will receive at least the minimum amount of NU needed to stake.
+ - Once all bidders have been assigned the minimum amount of NU, each bidder with a `bonus` will receive a portion of the remaining NU, distributed pro rata across all participants, taking into consideration only their bonus ETH amounts.
+ - If the resulting NU distributed to a bidder is above the maximum allowed NU to stake, then such a bidder has their bid partially refunded until the corresponding amount of NU is within the allowed limits.
 
-WorkLock offers specific advantages over ICO or airdrop as a distribution mechanism, chiefly: it selects for participants who are most likely to strengthen the network because they commit to staking and running nodes.
+Finally, if WorkLock participants use that stake-locked NU to run a node, the NU will eventually unlock and their escrowed ETH will be returned in full.
 
-The WorkLock begins with an open bidding period, during which anyone seeking to participate can send ETH to the WorkLock contract to be escrowed on-chain.
-At any time, WorkLock participants can cancel their bid to forgo NU and recoup their escrowed ETH immediately.
-Once the bidding period closes, the WorkLock contract doesn't accept more bids, although it still accepts cancellations during an additional time window.
-At the end of this cancellation period, stake-locked NU will be distributed according to the following principles:
 
- - Each bidder receives, at least, the minimum amount of NU needed to stake.
- - All bids will be greater or equal to the minimum allowed bid.
- - In addition to the minimum amount of NU, each bidder receives a portion of the remaining NU, distributed pro rata across all participants, taking into consideration their bid surplus with respect to the minimum bid.
- - If the resulting NU amount is above the maximum allowed NU to stake, then such a bidder has their bid partially refunded until the corresponding amount of NU is within the allowed limits.
+Hypothetical Bidding Scenarios
+------------------------------
 
-Finally, if WorkLock participants use that stake-locked NU to run a node, the NU will eventually unlock and their escrowed ETH will be returned in full.
+.. note::
+
+    To reduce complexity, calculations are performed in a step-wise manner which may lead to minor rounding differences
+    in the determined values.
+
+For each scenario, assume that:
+
+ #. WorkLock holds 280,000,000 NU tokens and the minimum bid is 15 ETH.
+ #. The minimum amount of NU required to stake is 15,000 NU and the maximum stake size is 4,000,000 NU.
```

Comment:
We are considering something in the range of 20-30M NU for the max stake, right?

Reply:
✔️ - Yep, that's still the plan (https://github.com/nucypher/nucypher/issues/1842).

Reworded "For each scenario, assume that..." to reinforce that the WorkLock properties specified are only hypothetical.